src/index.js

```diff
@@ -371,7 +372,7 @@ function EvolvClient(opts) {
    * @param {Object} [remoteContext] A map of data used for evaluating context predicates and analytics.
    * @param {Object} [localContext] A map of data used only for evaluating context predicates.
    */
-  this.initialize = function (uid, remoteContext, localContext) {
+  this.initialize = function (uid, remoteContext, localContext, pollForTimeUpdates) {
```

Comment:
I think this argument more appropriately belongs as an option in the client options:
https://github.com/evolv-ai/javascript-sdk/blob/67ed4990b6505312c9a4fcef7d7631c1753199ae/src/build-options.js#L8-L28

In most usages of `initialize()`, like with web-loader, `initialize()` is invoked automatically, and therefore you would have no opportunity to set `pollForTimeUpdates`.